.github/workflows/indent.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     - name: setup
       run: |
         ./contrib/utilities/download_clang_format
-        sudo apt-get install doxygen graphviz perl texlive-bibtex-extra
+        sudo apt-get update && sudo apt-get install doxygen graphviz perl texlive-bibtex-extra
```

Comment:
Not sure what kind of debian/ubuntu runs on that machine, and not sure about the non-ubuntu case at all, but isn't the preferred tool `apt` rather than `apt-get` nowadays?

Reply:
Lets see!

Reply:
@kronbichler OK. It has passed! Ready to be merged!